theme/static/js/hs-vue/subject-keywords-app.js

```diff
@@ -30,17 +33,18 @@ let subjKeywordsApp = new Vue({
             this.newKeyword = newKeywords.join(",").trim();
 
             if (this.newKeyword.trim() === "") {
+                this.error = "Your keyword was empty. Please try again";
```

Comment:
I don't think we need to handle this case with a message error.

Reply:
sounds good I will remove that